geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/ConnectCommandAcceptanceTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.geode.management.internal.cli.commands;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import org.apache.geode.test.dunit.standalone.VersionManager;
+import org.apache.geode.test.junit.categories.AcceptanceTest;
+import org.apache.geode.test.junit.categories.GfshTest;
+import org.apache.geode.test.junit.rules.gfsh.GfshExecution;
+import org.apache.geode.test.junit.rules.gfsh.GfshRule;
+import org.apache.geode.test.junit.rules.gfsh.GfshScript;
+
+@Category({AcceptanceTest.class, GfshTest.class})
+public class ConnectCommandAcceptanceTest {
+  @Rule
+  public GfshRule gfsh130 = new GfshRule(VersionManager.GEODE_130);
+
+  @Rule
+  public GfshRule gfshDefault = new GfshRule();
+
+  @Test
+  public void useCurrentGfshToConnectToOlderLocator() throws Exception {
+    GfshScript.of("start locator").execute(gfsh130);
+    GfshExecution connect = GfshScript.of("connect").expectFailure().execute(gfshDefault);
+
+    assertThat(connect.getOutputText()).contains("Cannot use a")
+        .contains("gfsh client to connect to this cluster.");
+  }
```

Comment:
I'd like to see tests for both cases of error result from `ConnectCommand`, i.e. `remoteVersion == null` and `!remoteVersion.equals(gfshVersion)`